### DISCUSSION TITLE
How to setup Tabby as default terminal in Linux?

### DISCUSSION THREAD

**Discussion by 403-html** [Eugeny/tabby#6266] [Q&A]
Hello, I'm wondering how you'd set default terminal in Ubuntu 22.04 to use Tabby?
I try to use `sudo update-alternatives --config x-terminal-emulator` but I don't get any other terminal emulator than default one to choose from:
```
There is only one alternative in link group x-terminal-emulator (providing /usr/bin/x-terminal-emulator): /usr/bin/gnome-terminal.wrapper
Nothing to configure.
```
I have tabby
```
~ » which tabby
/usr/bin/tabby
```

**Comment by icarocarmona** [comment]
@403-html try changing the [dconf](https://wiki.gnome.org/action/show/Projects/dconf?action=show&redirect=dconf) as follows:

```
gsettings set org.gnome.desktop.default-applications.terminal exec /usr/bin/tabby
gsettings set org.gnome.desktop.default-applications.terminal exec-arg "-x"

```
Thats works for me.

**Reply by vantienpham** [reply]
This works for me, v1.0.211 on Ubuntu 24.04.1. Thanks.

**Reply by soleroks** [reply]
it works, thanks!

**Comment by gauravagnihotristla** [comment]
which tabby
/usr/bin/tabby

# Register tabby
sudo update-alternatives --install /usr/bin/x-terminal-emulator x-terminal-emulator /usr/bin/tabby 50

# Select your terminal
sudo update-alternatives --config x-terminal-emulator
